sigstore-java/src/main/java/dev/sigstore/tuf/Updater.java

```diff
@@ -304,7 +317,6 @@ void updateTimestamp()
             localTimestamp.getSignedMeta().getVersion(), timestamp.getSignedMeta().getVersion());
       }
       if (localTimestamp.getSignedMeta().getVersion() == timestamp.getSignedMeta().getVersion()) {
-        trustedMetaStore.setTimestamp(localTimestamp);
```

Comment:
I wonder why I was doing this...

Reply:
I think I did this, in some weird refactor.